tests/phpunit/includes/bootstrap.php

```diff
@@ -14,24 +15,51 @@ function () use ( $_root_dir, $_tests_dir ) {
 
 		$GLOBALS['wp_plugin_paths'][ $symlink_dir_path ] = wp_normalize_path( dirname( $real_file_path ) );
 
+		// Tell WP where to find the themes.
+		register_theme_directory( "{$_root_dir}/tmp/themes" );
+		delete_site_transient( 'theme_roots' );
+
 		require_once $real_file_path;
 	}
 );
 
 define( 'WP_TESTS_PHPUNIT_POLYFILLS_PATH', $_root_dir . '/vendor/yoast/phpunit-polyfills/' );
 require_once $_root_dir . '/vendor/antecedent/patchwork/Patchwork.php';
 require_once $_tests_dir . '/includes/bootstrap.php';
+require_once $_root_dir . '/vendor/wpsyntex/wp-phpunit/UnitTests/functions.php';
 
 if ( ! defined( 'DIR_TESTROOT' ) ) {
 	define( 'DIR_TESTROOT', $_tests_dir );
 }
 
+if ( ! defined( 'PLL_TEST_EXT_THEMES_DIR' ) ) {
+	define( 'PLL_TEST_EXT_THEMES_DIR', "{$_root_dir}/tmp/themes/" );
+}
+
+if ( ! defined( 'PLL_TEST_EXT_PLUGINS_DIR' ) ) {
+	define( 'PLL_TEST_EXT_PLUGINS_DIR', "{$_root_dir}/tmp/plugins/" );
+}
+
+if ( ! defined( 'PLL_TEST_WP_DIR' ) ) {
+	define( 'PLL_TEST_WP_DIR', dirname( DIR_TESTROOT ) . '/wordpress/' );
+}
+
+if ( ! defined( 'PLL_TEST_THEMES_DIR' ) ) {
+	define( 'PLL_TEST_THEMES_DIR', PLL_TEST_WP_DIR . 'wp-content/themes/' );
+}
+
+if ( ! defined( 'PLL_TEST_PLUGINS_DIR' ) ) {
+	define( 'PLL_TEST_PLUGINS_DIR', PLL_TEST_WP_DIR . 'wp-content/plugins/' );
+}
```

Comment:
```suggestion
if ( ! defined( 'PLL_TEST_WP_DIR' ) ) {
	define( 'PLL_TEST_WP_DIR', dirname( DIR_TESTROOT ) . '/wordpress/' );
}

if ( ! defined( 'PLL_TEST_THEMES_DIR' ) ) {
	define( 'PLL_TEST_THEMES_DIR', "{$_root_dir}/tmp/themes/" );
}

if ( ! defined( 'PLL_TEST_PLUGINS_DIR' ) ) {
	define( 'PLL_TEST_PLUGINS_DIR', "{$_root_dir}/tmp/plugins/" );
}
```

Having 2 constants for plugins and themes directories is confusing. I propose never to use the default plugins and themes directories and always use ours.
This may also help us reminder not to use default themes. See what I made with Twenty Twenty Three in 9a8512075124bc81e250d119b47f687a5392a519.

Reply:
Didn't know about [those themes](https://github.com/WordPress/wordpress-develop/tree/trunk/tests/phpunit/data/themedir1). Done in [2ec97c9](https://github.com/polylang/polylang/pull/1581/commits/2ec97c910da990da759e9cd914bd95b523b09d46).